spec/models/case/sar/offender_spec.rb

```diff
@@ -254,6 +280,81 @@
     end
   end
 
+  describe '#subject_name' do
+    it 'is the same as subject_full_name' do
+      kase = create :offender_sar_case
+      expect(kase.subject_name).to eq kase.subject_full_name
+      kase.update_attribute(:subject_full_name, "Bob Hope")
+      expect(kase.subject_name).to eq 'Bob Hope'
+    end
+  end
+
+  describe '#subject_address' do
+    it 'returns a dummy string for now' do
+      pending "Will's PR"
```

Comment:
Just thinking it might be good to quote the ticket number or PR here - just incase someone else ends up picking this up.

Reply:
Thanks, I was being lazy! Will amend